core/src/main/java/org/infinispan/distribution/impl/L1ManagerImpl.java

```diff
@@ -140,7 +138,7 @@ public void addRequestor(Object key, Address origin) {
    }
 
    @Override
-   public Future<Object> flushCache(Collection<Object> keys, Address origin, boolean assumeOriginKeptEntryInL1) {
+   public Future<?> flushCache(Collection<Object> keys, Address origin, boolean assumeOriginKeptEntryInL1) {
```

Comment:
Well, I had to stop somewhere... the caller doesn't need any of the `CompletableFuture` features at the moment, when we get to making the interceptors async changing the `flushCache` signature will be easy.